automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSpec.scala

```diff
@@ -1,56 +1,155 @@
 package org.broadinstitute.dsde.workbench.leonardo
 
+import java.io.File
+import java.nio.file.Files
+import java.time.Instant
+
+import org.broadinstitute.dsde.workbench.{ResourceFile, WebBrowserSpec}
 import org.broadinstitute.dsde.workbench.config.AuthToken
 import org.broadinstitute.dsde.workbench.leonardo.ClusterStatus.ClusterStatus
+import org.broadinstitute.dsde.workbench.util.LocalFileUtil
+import org.openqa.selenium.WebDriver
 import org.scalatest.concurrent.Eventually
+import org.scalatest.selenium.WebBrowser
 import org.scalatest.time.{Minutes, Seconds, Span}
-import org.scalatest.{FreeSpec, Matchers}
+import org.scalatest.{FreeSpec, Matchers, ParallelTestExecution}
 
 import scala.util.Random
 
-class LeonardoSpec extends FreeSpec with Matchers with Eventually {
+class LeonardoSpec extends FreeSpec with Matchers with Eventually with WebBrowserSpec with WebBrowser with ParallelTestExecution with LocalFileUtil {
```

Comment:
Total test time is not much longer than 5 min because we create clusters for each in parallel